### PR TITLE
feat(CoGS): Add `shared-resources-usage` topic schema

### PR DESCRIPTION
**Pull request by rahul-kumar-saini**
### Overview
- We've added a new topic and will start producing data from Snuba to it soon
- This topic is how we will share CoGS data with the Data team

### Notes
- I'm not sure if adding this here will automatically start validating the messages on this topic or if there are any side effects at all

### CODE REVIEW

**Review by lynnagara — APPROVED** [review]

File: schemas/shared-resources-usage.v1.schema.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
```
Comment (lynnagara) [inline schemas/shared-resources-usage.v1.schema.json:2]: see the ci-failure. we enforce draft-07 so all the formats are the same

**Comment by fpacifici** [comment]
Thanks a lot @rahul-kumar-saini .
@lynnagara  The usage unit is an enum actually. What do we do normally? Do we store them just as strings or do we declare enums ?

**Comment by lynnagara** [comment]
No hard and fast rule but I'd go ahead an enumerate the values, I think there's no reason not to be strict here and it helps readers better understand what's expected on this topic

**Comment by rahul-kumar-saini** [comment]
@lynnagara I introduced `getsentry/super-big-consumers` as a valid consumer/producer, are there any side effects of doing this? I've also adjusted the string to an enum of valid usage unit types.

**Review by lynnagara — APPROVED** [review]

**Pull request merged**